pandas/tests/reshape/merge/test_merge_asof.py

```diff
@@ -1323,7 +1323,7 @@ def test_merge_index_column_tz(self):
                 "from_date": index[1:],
                 "abc": [2.46] * 3 + [2.19],
             },
-            index=pd.Index([1, 2, 3, 4]),
+            index=index[1:],
```

Comment:
can you make this explict (e.g. write it out)

Reply:
Also done, thanks for the review